tests/slow/src/test/scala/tests/feature/DefinitionCrossSlowSuite.scala

```diff
@@ -32,4 +32,33 @@ object DefinitionCrossSlowSuite
       _ = assertNoDiff(client.workspaceDiagnostics, "")
     } yield ()
   }
+
+  testAsync("2.13") {
```

Comment:
Can we abstract over this test so that it's clearer we test the same functionality on 2.11 and 2.13?